vortex-flatbuffers/src/lib.rs

```diff
@@ -146,17 +146,17 @@ pub trait WriteFlatBuffer {
 }
 
 pub trait WriteFlatBufferExt: WriteFlatBuffer + FlatBufferRoot {
-    /// Write the flatbuffer into a [`Buffer`].
-    fn write_flatbuffer_bytes(&self) -> Buffer;
+    /// Write the flatbuffer into a [`Bytes`].
+    fn write_flatbuffer_bytes(&self) -> Bytes;
```

Comment:
I don't know...

So FlatBuffers don't have any prescribed alignment, so long as the individual values are sufficiently aligned. e.g. if you store a int64 it must be 8 byte aligned.

So the vec that comes out of FlatBufferBuilder might not be 8 byte aligned. We could force it to be (with a possible copy), but the flatbuffer isn't wrong or broken. So it seems an odd API to force the copy when it may not be necessary.

Does that make sense?